test/powershell/Modules/Microsoft.PowerShell.Management/Test-Path.Tests.ps1

```diff
@@ -128,6 +128,14 @@ Describe "Test-Path" -Tags "CI" {
         Test-Path -Path $badPath -IsValid | Should -BeFalse
     }
 
+    It "Should return appropriate result for wildcard" {
+        if ($IsWindows) {
+            Test-Path -Path "C:\Program Files\*" -IsValid | Should -BeFalse
```

Comment:
I think a lot more tests are needed here to validate the logic. We'll probably need

+ More complex drive roots in Windows with both valid and invalid scenarios to verify
+ Both files and folder components containing valid and invalid scenarios
+ ADS specifiers on Windows on the last filename.

Reply:
Will add more tests